Reference/Config/customErrorPages/index.md

```diff
@@ -0,0 +1,81 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to implement custom error pages
+
+Since Umbraco is built upon Microsoft's .NET Framework and is using ASP.NET, you have several options when it comes to setting up custom error pages on your website.
```

Comment:
Would like a few sentences here, on what the article aims to do - What do you get out of reading this article?

Reply:
Added a sentence to explain the benefit of having custom errors set... hope it makes sense :)